### PR TITLE
Modules/Developer-Edition: Change - Manage auto update through wordpress base auto update.

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: modules/developer-edition/settings-page.php [hunk 1]
```diff
@@ -42,7 +43,10 @@ function () {
 				null,
 				static::PAGE_ID
 			);
-			register_setting( Module::SETTINGS_KEY, 'elementor_dev_auto_update' );
+
+			register_setting( Module::SETTINGS_KEY, 'elementor_dev_auto_update', [
```
Comment (matipojo) [inline modules/developer-edition/settings-page.php:47]: Check it the DB is not updated with an option `elementor_dev_auto_update`

**Review by matipojo — CHANGES_REQUESTED** [review]

File: modules/developer-edition/settings-page.php (hunk 1 above)
Comment (Nevoss) [reply modules/developer-edition/settings-page.php:47]: I added "return false" at the end of the sanitize_callback method and it does not save it in the DB.

File: modules/developer-edition/settings-page.php [hunk 2]
```diff
@@ -150,6 +154,29 @@ public function add_elementor_dev_modal_template() {
...
 	}
 
+	/**
+	 * Update auto update plugins option.
+	 *
+	 * @param $value
+	 *
+	 * @return false
+	 */
+	public function update_auto_update( $value ) {
```
Comment (matipojo) [inline modules/developer-edition/settings-page.php:164]:
```suggestion
	public function sanitize_update_auto_update( $value ) {
```

File: modules/developer-edition/settings-page.php [hunk 3]
```diff
@@ -150,6 +154,29 @@ public function add_elementor_dev_modal_template() {
...
+		$auto_updates = (array) get_site_option( 'auto_update_plugins', [] );
```
Comment (matipojo) [inline modules/developer-edition/settings-page.php:165]:
```suggestion
		$auto_updates = get_site_option( 'auto_update_plugins', [] );
```

File: modules/developer-edition/views/settings-page-auto-update.php [hunk 4]
```diff
@@ -1,15 +1,25 @@
 <?php
+
+use ElementorDev\Bootstrap;
+
 if ( ! defined( 'ABSPATH' ) ) {
 	exit;
 }
+
+$is_checked = false;
+$auto_updates = (array) get_site_option( 'auto_update_plugins', [] );
```
Comment (matipojo) [inline modules/developer-edition/views/settings-page-auto-update.php:10]:
```suggestion
$auto_updates = get_site_option( 'auto_update_plugins', [] );
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: modules/developer-edition/settings-page.php (hunk 3 above)
Comment (Nevoss) [reply modules/developer-edition/settings-page.php:165]: For empty string from the DB.

File: modules/developer-edition/views/settings-page-auto-update.php (hunk 4 above)
Comment (Nevoss) [reply modules/developer-edition/views/settings-page-auto-update.php:10]: Same as above.

**Review by matipojo — APPROVED** [review]

**Pull request merged**